src/index.ts

```diff
@@ -86,6 +86,7 @@ export {
 
 // enums
 export { BatchType } from './bulk/common';
+export { GSSAPICanonicalizationValues } from './cmap/auth/gssapi';
```

Comment:
Sorry, I had edited my comment after I posted so probably was missed, I think we should not make this plural, and we need **both** the enum export here (thanks!) and type export that was here below.

Here's an example of the code that ensuring both exports supports
```ts
import { GSSAPICanonicalizationValue } from 'mongodb';

function processKerberosOption(x: GSSAPICanonicalizationValue) { 
  // The same symbol from the one import above is both a type annotation
  // And a value that can be used as an enum
  x = GSSAPICanonicalizationValue.forwardAndReverse;
}
```

Reply:
Updated to `GSSAPICanonicalizationValue` and verified the single export works as-is.